css/style.css

```diff
@@ -96,6 +96,10 @@ html {
   font-weight: bold;
 }
 
+.dates-p {
+  margin: 0px;
+}
```

Comment:
I wonder if this is really the best CSS practice, but honestly the rest of the styles are pretty janky so this should be fine for now.